exercises/palindrome-products/example.js

```diff
@@ -14,16 +14,16 @@ module.exports = function Palindromes(options) {
     for (var i = minFactor; i <= maxFactor; i++) {
       for (var j = minFactor; j <= maxFactor; j++) {
         var result = i * j;
-        if ( ! this.isPalindrome(result) ) { continue; }
+        if ( !this.isPalindrome(result) ) { continue; }
 
         var newFactor = [i, j].sort();
 
-        if (palindromes[result] === undefined) {
+        if ( !Array.isArray(palindromes[result])) {
```

Comment:
Hmm.  It seems like if there is a leading space, there should be a trailing one?  Or none at all?